scripts/mochacfg.js

```diff
@@ -1,3 +0,0 @@
-['.png', '.gif'].forEach(extension => {
```

Comment:
I think you can also delete the `adapt-enzyme-to-react-16.js` file in `scripts`

Reply:
Done